resources/views/admin/chatrooms/categories/edit.blade.php

```diff
@@ -77,4 +76,18 @@
             </form>
         </div>
     </div>
+
+@endsection
+
+@section('scripts')
+    <script src="https://code.jquery.com/jquery-3.7.0.min.js"
```

Comment:
You can add this one in the includes so it is not repeating

Reply:
I moved it into layouts file.